var/spack/repos/builtin/packages/cryptopp/package.py

```diff
@@ -25,11 +25,29 @@ class Cryptopp(MakefilePackage):
     version('5.6.2', sha256='5cbfd2fcb4a6b3aab35902e2e0f3b59d9171fee12b3fc2b363e1801dfec53574')
     version('5.6.1', sha256='98e74d8cb17a38033354519ac8ba9c5d98a6dc00bf5d1ec3c533c2e8ec86f268')
 
+    variant('shared', default=True,  description="Build shared object versions of libraries.")
+    variant('sse',    default=True,  description="Build with instruction vectorization enabled.")
```

Comment:
We should probably use Spack's builtin microarch detection instead of a variant, @alalazo or @becker33 can show you the correct way to use it.

Reply:
I agree with @adamjstewart Since support for microarchitecture detection has been added I would say that the idiomatic way to activate SIMD support is to inspect the target. That has the benefit of reducing the number of variants that a user needs to deal with.